docs/onboarding_toolkit.md

```diff
@@ -1 +1,172 @@
-#  Onboarding toolkit
+# Ansible Project Onboarding
+
+Welcome to the Ansible ecosystem of projects! This document guides you 
+through the steps to take to become part of the Ansible ecosystem.
+
+# Why be part of the Ansible ecosystem
+Being a part of the Ansible ecosystem means you are tapping into a large and
+thriving community of passionate people working and volunteering in Ansible.
+
+The major benefits of becoming part of the Ansible ecosystem include:
+
+* A presence on docs.ansible.com/ecosystem for your project documentation,
+ as well as being part of the official Ansible ReadTheDocs space.
+* A presence on the (future) Ansible.com to introduce your project.
+* Dedicated communications channels (chat and forum) to communicate with your
+community and attract other community members to your project.
+
+
+
+# Setting up your GitHub repository
+
+The Ansible ecosystem projects exist in the following two GitHub repositories:
+
+* ansible - Access is controlled by Red Hat
+* ansible-community - Access is controlled by the community, 
+under Red Hat sponsorship
+
+We recommend most Ansible projects use the ansible-community GitHub repository.
+Repository transfers are available if your project already exists somewhere else in GitHub.
+
+For new projects, use the [project-template](https://github.com/ansible-community/project-template) to 
+create a project with the default files.
+
+## Other repository settings
+
+* Open up issues and PRs for task tracking and community contributions.
+* Disable the GitHub discussions. Use the 
+[Ansible Community forum](https://forum.ansible.com/) for community and project discussions.
+* Disable the repo wiki for project documentation. See the sections below on 
+documentation and using the Ansible forum.
+
+## Common files
```

Comment:
Maybe call this "how to submit your project for inclusion" and add a point on how to submit after all the changes/requirements?